inst/help/MixedModelsGLMM.md

```diff
@@ -0,0 +1,152 @@
+Generalized Linear Mixed Models
+===
+
+Generalized Linear Mixed Models allow you to model a linear relationship between one or more explanatory variable(s) and a continuous dependent variable in cases where the observations are not independent, but clustered given one or several random effects grouping factors (e.g., repeated measures across participants or items, children within schools). They are generalization of Linear Mixed Models and allow to model response variables that are not continous using a different likelihoods and link functions.
+
+### Assumptions
+- Linearity and additivity: The response variable is linearly related to all predictors and the effects of the predictors are additive on the linear scale.
+- Independence of errors: The errors are uncorrelated with each other after taking the model (i.e., fixed effects and random effects structure) into account.
+- Homoscedasticity: The error variance of each predictor is constant across all values of that predictor.
+- Normality of errors: The errors are normally distributed with mean zero.
```

Comment:
same thing here as with bayesian: linearity only with link function, and errors are distributed according to distributional family (link-inverse linear combination of the predictors is the expectation of this distribution)
y ~ DistributionalFamily(linkinv(x*b), dispersion)

Reply:
yes, you are right. I thought that we had updated that but it must've gotten lost somewhere. Thanks for pointing it out.

Reply:
thanks for noticing the errors in the help files, I updated them with one of the commits